Resources/SteamLanguage/emsg.steamd

```diff
@@ -1883,8 +1933,8 @@ enum EMsg
 	SLCSharedLibraryChanged = 9408;
 
 	RemoteClientBase = 9500;
-	RemoteClientAuth = 9500;
-	RemoteClientAuthResponse = 9501;

```

Comment:
Removing these without marking them as `removed` or renamed?

Reply:
Theye're marked as removed anyway, does it matter to leave old one in place with removed+renamed comment?

Reply:
Just in case someone happened to be using that EMsg so that we don't break them.

Reply:
`removed` entries are not generated into C#

Reply:
`obsolete` generates C#, `removed` does not.